app/models/person.rb

```diff
@@ -63,6 +63,10 @@ def location
     location_association || build_location
   end
 
+  def completed_activities
+    actions.joins(:activity).pluck("activities.template_id")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.